packages/zoe/test/unitTests/contractSupport/test-ratio.js

```diff
@@ -351,12 +369,12 @@ test('ratio - quantize', t => {
     [12345n, 12345n, 10n ** 15n, 10n ** 15n],
 
     [12345n, 123n, 100365854n, 10n ** 6n],
-    [12345n, 123n, 10036586n, 10n ** 5n],
+    [12345n, 123n, 10036585n, 10n ** 5n],
```

Comment:
now each value here is the simple rounding of the line above divided by 10. (`100365854 / 10` should round to `10036585` not `100365856`)